pages/Configuring/Expanding-functionality.md

```diff
@@ -7,12 +7,12 @@ The second, socket2, sends events for certain changes / actions and can be used
 to react to different events. See its description
 [here](../../IPC/).
 
-## Example script
+# Example script
 
 This bash script will change the outer gaps to 20 if the currently focused
 monitor is DP-1, and 30 otherwise.
 
-```bash
+```sh
```

Comment:
Why is this one changed, and further down the PR it changes `sh` to `bash`?

Reply:
No clue, but we should probably keep them all as sh except that one xdg-desktop-portal-... example.

Reply:
Update - that one uses ```sh because the script below used /bin/sh and not /bin/bash.